model/cat/pit_segments/operations.smithy

```diff
@@ -21,3 +20,17 @@ operation CatPitSegments {
     input: CatPitSegments_Input,
     output: CatPitSegments_Output
 }
+
+@externalDocumentation(
+    "API Reference": "https://opensearch.org/docs/latest/search-plugins/point-in-time-api/"
+)
+@xOperationGroup("cat.all_pit_segments")
```

Comment:
Ah didn't realise we had a hard rule on that, can split it back out to a seperate file if you want

Reply:
It's not a hard rule. I can see why you wanna do it this way because these groups share a lot of common components.

When I regenerated this repo to include the bodies, they will moved by that rule anyway :P